pkg/config/host/names.go

```diff
@@ -59,6 +59,11 @@ func MoreSpecific(a, b Name) bool {
 	return len(a) > len(b)
 }
 
+// isWildCarded checks if the string starts with a wildcard character '*'.
+func isWildCarded(host Name) bool {
```

Comment:
can this replace with `strings.HasPrefix(host, "*")` ?

Reply:
sounds good, should be same performance. Updated it. Thanks for reviewing.